db/schema.rb

```diff
@@ -1,101 +1,96 @@
+  create_table "comments", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.text "comment"
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  create_table "cost_codes", id: :integer, force: :cascade do |t|
-    t.string "name", limit: 255, null: false
+  create_table "cost_codes", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.string "name", null: false
   end
 
-  create_table "events", id: :integer, force: :cascade do |t|
-    t.integer  "asset_id",   limit: 4, null: false
-    t.integer  "state_id",   limit: 4, null: false
+  create_table "events", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.integer "asset_id", null: false
+    t.integer "state_id", null: false
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  create_table "pipeline_destinations", id: :integer, force: :cascade do |t|
-    t.string "name", limit: 255
+  create_table "pipeline_destinations", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.string "name"
   end
 
-  create_table "states", id: :integer, force: :cascade do |t|
-    t.string   "name",       limit: 255, null: false
+  create_table "states", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.string "name", null: false
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  create_table "workflows", id: :integer, force: :cascade do |t|
-    t.string   "name",             limit: 255,                 null: false
-    t.boolean  "has_comment",                  default: false, null: false
+  create_table "workflows", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.string "name", null: false
+    t.boolean "has_comment", default: false, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.boolean  "reportable",                   default: false, null: false
-    t.integer  "turn_around_days", limit: 4
-    t.integer  "initial_state_id", limit: 4
-    t.boolean  "active",                       default: true,  null: false
-    t.boolean  "qc_flow",                      default: false, null: false
-    t.boolean  "cherrypick_flow",              default: false, null: false
+    t.boolean "reportable", default: false, null: false
+    t.integer "turn_around_days"
+    t.integer "initial_state_id"
+    t.boolean "active", default: true, null: false
+    t.boolean "qc_flow", default: false, null: false
+    t.boolean "cherrypick_flow", default: false, null: false
+    t.index ["initial_state_id"], name: "fk_rails_e3fad0d986"
   end
 
-  add_index "workflows", ["initial_state_id"], name: "index_workflows_on_initial_state_id", using: :btree
-
-  add_foreign_key "assets", "asset_types"
-  add_foreign_key "assets", "batches"
-  add_foreign_key "assets", "comments"
-  add_foreign_key "assets", "workflows"
```

Comment:
Huh, usually this one happens the other way round, due to early versions of Rails not supporting foreign keys fully, so we'd create them manually, but they'd be missing from the schema.